src/plugins/vis_builder/public/application/utils/get_top_nav_config.tsx

```diff
@@ -46,22 +46,24 @@ export interface TopNavConfigParams {
   savedVisBuilderVis: VisBuilderVisSavedObject;
```

Comment:
how come this file has the old license? i believe this is a new plugin right?

Reply:
i think it's because this file is very similar to the get_top_nav_config.tsx in visualize plugin?

Reply:
yep - mostly copied.

Reply:
probably we should update it